segmentation/include/pcl/segmentation/grabcut_segmentation.h

```diff
@@ -438,39 +438,39 @@ namespace pcl
       inline bool
       isSource (vertex_descriptor v) { return (graph_.inSourceTree (v)); }
       /// image width
-      std::uint32_t width_;
+      std::uint32_t width_{0};
       /// image height
-      std::uint32_t height_;
+      std::uint32_t height_{0};
       // Variables used in formulas from the paper.
       /// Number of GMM components
-      std::uint32_t K_;
+      std::uint32_t K_{0};
```

Comment:
```suggestion
      std::uint32_t K_;
```
Set in the constructor

Reply:
Missed this one?